application/helpers/common_helper.php

```diff
@@ -172,28 +172,32 @@ function isStandardTemplate($sTemplateName)
 */
 function getSurveyList($bReturnArray = false)
 {
-    static $cached = null;
+    static $static = null;
     $bCheckIntegrity = false;
     $timeadjust = getGlobalSetting('timeadjust');
     App()->setLanguage((isset(Yii::app()->session['adminlang']) ? Yii::app()->session['adminlang'] : 'en'));
     $surveynames = array();
 
-    if (is_null($cached)) {
+    if (is_null($static)) {
+        $criteria = new CDBCriteria();
+        $criteria->select = array(
+            'sid','owner_id','active','expires'
+        );
         $surveyidresult = Survey::model()
             ->permission(Yii::app()->user->getId())
-            ->with('languagesettings')
-            ->findAll();
+            ->with('defaultlanguagetitle')
```

Comment:
I like this idea.

Reply:
With a low quality connection between SQL and WEB server : you really see a big difference.

Must redo for master here …